docs/book/src/clusterctl/commands/alpha-rollout.md

```diff
@@ -0,0 +1,51 @@
+# clusterctl alpha rollout
+
+The `clusterctl alpha rollout` command manages the rollout of a cluster-api resource. It consists of several sub-commands which are documented below. 
+
+<aside class="note">
+
+<h1> Valid Resource Tpes </h1>
+
+Currently, only the following cluster API resources are supported by the rollout command:
+
+- machinedeployment
+
+</aside>
+
+### Restart 
+
+Use the `restart` sub-command to force an immediate rollout. For example, here the MchineDeployment `my-md-0` will be immediately rolled out:
+
+```
+clusterctl alpha rollout restart machinedeployment/my-md-0
+```
+
+### Undo
+
+Use the `undo` sub-command to rollback to an ealier revision. For example, here the MachineDeployment `my-md-0` will be rolled back to revison number 3. If the `--to-revision` flag is omited, the MachineDeployment will be rolled back to the revision immediately preceding the current one. 
```

Comment:
should we document what happen if the requested revision/the immediate preceding revision does not exists?